cmd/cstor-pool-mgmt/pool/v1alpha2/pool_utils.go

```diff
@@ -173,3 +180,132 @@ func getDeviceType(r apis.RaidGroup) string {
 	}
 	return DeviceTypeEmpty
 }
+
+// getBlockDeviceClaimList returns list of block device claims based on the
+// label passed to the function
+func getBlockDeviceClaimList(key, value string) (
+	*blockdeviceclaim.BlockDeviceClaimList, error) {
+	namespace := env.Get(env.Namespace)
```

Comment:
Let's have one global variable namespace. Setting namespace can be done by caller package or init fn.

Reply:
I will take this thing as different PR (like before starting the controller we will fetch and update the global variable and we will use it).